table/server/master/src/main/java/alluxio/master/table/Database.java

```diff
@@ -353,6 +338,30 @@ public SyncStatus sync(JournalContext context) throws IOException {
     return builder.build();
   }
 
+  private void loadFilterSpec() throws IOException {
+    if (mConfigPath.equals(CatalogProperty.DB_CONFIG_FILE.getDefaultValue())) {
+      return;
+    }
+    if (!Files.exists(Paths.get(mConfigPath))) {
+      throw new FileNotFoundException(mConfigPath);
+    }
+    ObjectMapper mapper = new ObjectMapper();
+    File configFile = new File(mConfigPath);
+    if (configFile.length() == 0) {
+      mFilterSpec = UdbFilterSpec.empty();
+      return;
+    }
+    try {
+      mFilterSpec = mapper.readValue(configFile, UdbFilterSpecJsonSource.class).getUdbFilterSpec();
+    } catch (JsonProcessingException e) {
+      LOG.error("Failed to parse UDB config file {}, likely syntax error", mConfigPath, e);
+      throw new IOException(String.format("Failed to parse UDB config file %s", mConfigPath), e);
```

Comment:
typically LOG and throw is anti-pattern as it may cause the same error logged multiple times. 
please be very sure this is required or let's just throw the IOE and leave the caller side to decide if this shall be logged

Reply:
Fixed in 14411fbc2c1e88fe8ad8993bba095b0e13f4d213